lib/controllers/ServiceFabrikApiController.js

```diff
@@ -168,16 +168,23 @@ class ServiceFabrikApiController extends BaseController {
   }
 
   getInfo(req, res) {
+    let dockerMissingImages = [];
     return docker
       .getMissingImages()
-      .then(missingImages => res
-        .status(200)
-        .json({
-          name: this.serviceBrokerName,
-          api_version: this.constructor.version,
-          ready: _.isEmpty(missingImages)
-        })
-      );
+      .then(missingImages => dockerMissingImages = missingImages)
+      .catch((err) => {
+        logger.info('error occurred while fetching docker images', err);
+        dockerMissingImages = ['Error occurred while fetching docker images'];
```

Comment:
better to leave as empty array or null object itself. As an error statement in array of images doesn't look good. If someone parses it and displays, then gets an image with that statement, which is invalid!

Reply:
Fixed it in better way.